backend/README.md

```diff
@@ -27,14 +27,18 @@ the regular base application.
+|----------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| POSTGRES_USER                          | Name of the user, needed to login to the postgres database                                                                                                                                                                                                                                                                        |
+| POSTGRES_PASSWORD                      | Password of the user, needed to login to the postgres database                                                                                                                                                                                                                                                                    |
+| POSTGRES_HOST                          | Location of the postgres database                                                                                                                                                                                                                                                                                                 |
+| POSTGRES_DB                            | Name of the postgres database                                                                                                                                                                                                                                                                                                     |
+| API_HOST                               | Location of the API root                                                                                                                                                                                                                                                                                                          |
+| CLIENT_ID                              | [Client id](https://learn.microsoft.com/nl-nl/entra/identity-platform/v2-protocols)                                                                                                                                                                                                                                               |
+| CLIENT_SECRET                          | Client's secret is your personal secret key for authentication                                                                                                                                                                                                                                                                    |
```

Comment:
Maybe refer that this is found in the Entra ID admin center? (Microsoft documentation)

Reply:
https://github.com/SELab-2/UGent-3/pull/357/commits/5440a8555939b3e2f5ac735723e8e0d8abc2ac03